src/envelopeCommands.cpp

```diff
@@ -657,3 +657,40 @@ void postSelectMultipleEnvelopePoints(int command) {
 		translate_plural("{} point selected", "{} points selected", count),
 		count));
 }
+
+void cmdMoveSelEnvelopePoints(Command* command) {
+	TrackEnvelope* envelope;
+	double offset{0.0};
+	tie(envelope, offset) = getSelectedEnvelopeAndOffset();
+	if(!envelope || !currentEnvelopePoint || !shouldReportTimeMovement()) {
+		Main_OnCommand(command->gaccel.accel.cmd, 0);
+		return;
+	}
+	double oldPos {0.0};
+	GetEnvelopePointEx(envelope, currentAutomationItem, *currentEnvelopePoint, &oldPos, nullptr, nullptr, nullptr, nullptr);
+	oldPos += offset;
+	Main_OnCommand(command->gaccel.accel.cmd, 0);
+	double newPos{0.0};
+	GetEnvelopePointEx(envelope, currentAutomationItem, *currentEnvelopePoint, &newPos, nullptr, nullptr, nullptr, nullptr);
+	newPos += offset;
+	ostringstream s;
+	if(lastCommand != command->gaccel.accel.cmd) { 
+		const char* name = kbd_getTextFromCmd(command->gaccel.accel.cmd, nullptr);
+		const char* start;
+		// Skip the category before the colon (if any).
+		for (start = name; *start; ++start) {
+			if (*start == ':') {
+				name = start + 2;
+				break;
+			}
+		}
+		s<<name << " ";
```

Comment:
We now have this action name + category stripping code in three places. I think it'd be good to unify this. Perhaps we could move the bulk of reportActionName into an appendActionName, outputActionName, formatActionName or similar which takes an ostringstream reference. Or we could have getActionName which returns a string, but that seems like wasteful copying given that we're always going to output to an ostringstream anyway.

Reply:
I changed reportActionName to getActionName, returning const char*.  Is that okay?